modules/ROOT/partials/commands/importword-cmds.adoc

```diff
@@ -0,0 +1,11 @@
+[cols="1,3",options="header"]
+|===
+|Command |Description
+|ImportWord |Opens a file picker, that allows content from either a `.docx` or `.dotx` file to be imported into the editor.
+|===
+
+.Example
+[source,js]
+----
+tinymce.activeEditor.execCommand('ImportWord');
+----
```

Comment:
Add, please, the Note that the ImportWord command should be initiated through a user activation, such as a button click.

Reply:
```suggestion
----

[NOTE]
To activate the `ImportWord` command, users are required to click on the "Import to Word" button. Upon clicking the button, the `ImportWord` command is triggered.
```

Reply:
**That is not pretty what I intended.**
Unlike other commands, this one can not be directly invoked via a script at the top level or from the browser console. Instead, it requires a specific user action within the user interface (UI), such as clicking a toolbar button (it may be any custom button, not necessarily the 'ImportWord' registered button). To ensure proper execution, the command should be called within the scope of an event handler of specific user interaction.